_NscLib/CMakeLists.txt

```diff
@@ -20,16 +20,18 @@ add_library(nsclib
         Precomp
 )
 
+target_include_directories(nsclib PRIVATE . ${CMAKE_CURRENT_BINARY_DIR})
+
 if (WIN32 OR WIN64)
     add_custom_command(
-            OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/NscParser.cpp ${CMAKE_CURRENT_SOURCE_DIR}/NscParser.hpp
+            OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/NscParser.cpp ${CMAKE_CURRENT_BINARY_DIR}/NscParser.hpp
```

Comment:
Why `CMAKE_CURRENT_BINARY_DIR`?

Reply:
I just feel like that's where build artifacts and generated files should go.  The files are ignored as far as git goes.  If you prefer to keep the generated files in the source tree that's fine by me.  Lmk and I'm happy to revert that.

Reply:
It is not a build artifact though, it is intermediate code used for compilation.

Reply:
Yes, I understand.  They are generated files that are specifically included in [gitignore](https://github.com/nwneetools/nwnsc/blob/master/_NscLib/.gitignore) and used for compilation.  So why put those files into the source tree at all?  It's not like anyone could or should ever edit them or it doesn't build anymore.  But like I said I'm happy to revert it, I don't really have any strong feelings about this.

Reply:
That commit's been dropped.  It's not worth even this much discussion.